README.md

```diff
@@ -1,51 +1,66 @@
 # Resinhup
 Tool for resin host OS updates.
 
```

Comment:
Might be good to add a short description of the workflow as an intro. i.e. something like "This tool is a docker image that downloads an update bundle that replaces the HostOS for a Resin device, updating both boot partition and rootfs (using an inactive partition)".